testing/go/sequences_test.go

```diff
@@ -403,6 +403,38 @@ func TestSequences(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name: "nextval() with double-quoted identifiers",
```

Comment:
Do we not have any tests of using nextval as a default value?

Reply:
We do have tests using `nextval()` as a column default value. I tested here with direct selecting just for simplicity since it repros the problem as well. 

The other issue I saw with `nextval()` that I mentioned to you early turns out to only be triggered with `::regclass` is used to cast the value of the string identifier. I'm tracking the details for that one in https://github.com/dolthub/doltgresql/issues/850 and will have some new tests for column defaults with that one.